sound-bubble/imports/ui/components/Song.jsx

```diff
@@ -103,8 +103,12 @@ class Song extends Component {
   }
 
   render() {
-    const { track, song, groupSong, home } = this.props;
+    const { track, song, groupSong, home, filterKey, user } = this.props;
     if (song) {
+      if(home && !(song.name.toLowerCase().includes(filterKey.toLowerCase())) && !(user.display_name.toLowerCase().includes(filterKey.toLowerCase()))){
```

Comment:
User can be null here. I tried leaving something in the filter and switch to another group then switch back, it threw an error. Maybe add user as part of the && check before using it. 
Also, how about we add artist names to also be filterable here? Should be quite easy.

Reply:
artist can be filterable. I go check the bug